Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -41,14 +41,17 @@ final class InitializerCommand: ParsableCommand {
   @Flag(name: .shortAndLong, default: true, inversion: .prefixedEnableDisable, help: typeOnlyHelp)
   var typeOnly: Bool
 
+  @Option(parsing: .upToNextOption, help: argumentNameHelp)
+  var argumentName: [String]
```

Comment:
Why not call it `paramaterName`? Also should it be plural "names" given that it's an array?

Reply:
So this can be used from the command line like `--argument-name Some` or with multiple parameters like `--argument-name Some Another`. You're right that parameter name is probably a better name, I'll update.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/22/commits/91dfffece62faf487cefe7623940f2b6188560bf